clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/edda/InstanceLoadBalancerState.groovy

```diff
@@ -16,21 +16,59 @@
 
 package com.netflix.spinnaker.clouddriver.aws.model.edda
 
+import com.netflix.spinnaker.clouddriver.model.HealthState
 import groovy.transform.CompileStatic
 import groovy.transform.EqualsAndHashCode
-import groovy.transform.Immutable
 
-@Immutable
 @CompileStatic
 @EqualsAndHashCode(cache = true)
 class InstanceLoadBalancerState {
-  String instanceId
-  String loadBalancerName
-  String state
-  String reasonCode
-  String description
+  final String instanceId
+  final String loadBalancerType
+  final String loadBalancerName
+  final String state
+  final String reasonCode
+  final String description
+
+  final HealthState healthState
```

Comment:
this is now part of this object representation so the front end can use it instead of interpreting `state` (InService on classic load balancers, vs healthy on ELBv2)